kong/plugins/hmac-auth/access.lua

```diff
@@ -98,7 +99,8 @@ local function load_secret(username)
       local keys, err = dao.hmacauth_credentials:find_by_keys { username = username }
       local result
       if err then
-        return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+        ngx_log(ngx.ERR, err)
+        return responses.send_HTTP_INTERNAL_SERVER_ERROR("Internal system error")
```

Comment:
Like I said: don't log, it would log twice. I meant `send_HTTP_INTERNAL_SERVER_ERROR` logs for you, and override the error with "Internal Server Error", so replace your string by `err`.